CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Welcome to the Momento Go-Redis contributing guide :wave:
+
+Thank you for taking your time to contribute to our Momento `go-redis` wrapper!
+<br/>
+This guide will provide you information to start your own development and testing.
+<br/>
+Happy coding :dancer:
+<br/>
+
+## Requirements :coffee:
+
+- Go 1.2 or above is required https://go.dev/doc/install
```

Comment:
According to the `CONTRIBUTING` doc in the SDK, our minimum supported Go version is currently 1.18.